kani-compiler/src/main.rs

```diff
@@ -128,13 +137,37 @@ struct KaniCallbacks {}
 /// Use default function implementations.
 impl Callbacks for KaniCallbacks {}
 
+/// The Kani root folder has all binaries inside bin/ and libraries inside lib/.
+/// This folder can also be used as a rustc sysroot.
+fn kani_root() -> PathBuf {
+    match env::current_exe() {
+        Ok(mut exe_path) => {
+            // Current folder (bin/)
```

Comment:
Isn't there a better way to move to the root folder? This looks like it's only good to go from `bin/` to the root folder.

Reply:
Unfortunately, this does rely on the structure of the root folder and the location of `kani-compiler`. This is the logic we already used for bundled Kani and it's very similar to `rustc` logic. One thing I just noticed that `rustc` does is to canonicalize the path to follow symbolic links. I'll add this here.

As a fallback plan, I kept the command argument to set sysroot and the library path in case we need to override those values.

Do you have any other suggestion?

Reply:
Not really, thanks for the detailed explanation!